Build/Laptop/EMail.cc

```diff
@@ -1916,7 +1917,7 @@ static void HandleIMPCharProfileResultsMessage(void)
 		// have to place players name into string for first record
 		if (i == 0)
 		{
-			wchar_t zTemp[512];
+			wchar_t zTemp[NAME_LENGTH];
```

Comment:
zName has the length of NAME_LENGTH too, that's why I chose it. So, if zName overflows, we will too, but it's probably good to see it when it happens so we notice and fix it.